sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,12 +1,6 @@
 # Release History
 
-## 12.17.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 12.17.0-beta.2 (2022-05-24)
```

Comment:
Any dependency updates with this release?

Reply:
None outside of storage. Our last patch picked up the latest core, and there won't be a new one until the scheduled release beginning of june

Reply:
Curious why the pom wasn't already updated to the correct version? We ran the version update script, right?

Reply:
Is it because this is a GA release?  We have to override the versions set by the bot that sets the .NET versions when doing a GA release, because it defaults to beta versions.